GVFS/GVFS.Common/NuGetUpgrader/NuGetUpgrader.cs

```diff
@@ -349,6 +358,16 @@ private static bool TryGetPersonalAccessToken(string gitBinaryPath, string crede
             return gitProcess.TryGetCredentials(tracer, credentialUrl, out string username, out token, out error);
         }
 
+        private string ReplaceTokens(string src)
+        {
+            string logFilePath = ProductUpgraderInfo.GetLogDirectoryPath();
+            string timeStamp = DateTime.Now.ToString("yyyyMMdd_HHmmss");
+
+            string dst = src.Replace("{logpath}", logFilePath)

```

Comment:
Style suggestion:

Use named constants for `"{logpath}"` and `"{timestamp}"` (perhaps put them in a class that has the well known manifest tokens?)